server/src/game/Game.cpp

```
@@ -162,8 +175,8 @@ void game::Game::stop() {
  * frame rate.
  */
 void game::Game::gameLoop() {
-  std::this_thread::sleep_for(
-      std::chrono::seconds(3));  // TODO: remove when lobby
+  std::this_thread::sleep_for(std::chrono::seconds(3));
+
   queue::GameStartEvent startEvent;
```

Clang (14.0.6)

[warning] variable 'startEvent' is not initialized

(cppcoreguidelines-init-variables)